.eslintrc.js

```diff
@@ -10,10 +10,42 @@ module.exports = {
     },
 
     {
-      files: ['*.test.js'],
+      files: ['*.ts'],
+      extends: ['@metamask/eslint-config-typescript'],
+      // TODO: Migrate to our shared config
+      settings: {
+        'import/resolver': {
+          // Try resolving imports using TypeScript's module resolution
+          // algorithm, then Node's
```

Comment:
What is this for?

Reply:
It is for the JS compatibility tests, but it's so that when importing a TypeScript module in the JavaScript tests, the `import` ESLint plugin doesn't get confused. By default, when it resolves imports, the import plugin will only look for `.js` files. This makes it so that it'll look for `.ts` files too.

Reply:
Right. It just seems easier to me to avoid that completely instead, and not write JavaScript tests.

Reply:
Ahh, I see what you mean. Okay, I'll give that a shot.

Reply:
Tests consolidated in [c05ed66](https://github.com/MetaMask/eth-json-rpc-infura/pull/58/commits/c05ed6652798699ec38446d67548c83db47aec41).